ci-operator/jobs/openshift/oadp-operator/openshift-oadp-operator-master-presubmits.yaml

```diff
@@ -1047,7 +1047,7 @@ presubmits:
           secretName: result-aggregator
     trigger: (?m)^/test( | .* )4.12-operator-e2e-azure,?($|\s.*)
   - agent: kubernetes
-    always_run: true
+    always_run: false
```

Comment:
I think we are missing `run_if_changed` in e2e aws, azure and unit test for 4.12 (for gcp not, as it should disabled for now)

Should we fix this in this PR?

Reply:
Could do. sure